### PR TITLE
Propose JuliaAstro project

### PR DESCRIPTION
**Pull request by giordano**
Trying to bring @dpshelio to the dark side :smiling_imp:

### CODE REVIEW

File: _projects/2019/juliaastro/astroimages.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ - Astronomy
+ - Julia
+difficulty: medium
+issues:
+ - https://github.com/JuliaAstro/AstroImages.jl/issues/1
+ - https://github.com/JuliaAstro/AstroImages.jl/issues/2
+ - https://github.com/JuliaAstro/FITSIO.jl/issues/90
+mentors:
+ - giordano
+ - dpshelio
```
Comment (bsipocz) [inline _projects/2019/juliaastro/astroimages.md]: treason :trollface:

File: _projects/2019/juliaastro/astroimages.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+name: Visualization of astronomical images in Julia
+desc: Expand the capabilities of the AstroImages.jl package
+requirements:
+ - Astronomy
```
Comment (bsipocz) [inline _projects/2019/juliaastro/astroimages.md]: is astronomy required though for this project?

Comment (giordano) [reply _projects/2019/juliaastro/astroimages.md]: Well, not astronomy as a whole but at least familiarity with FITS files would be appreciated.

Comment (bsipocz) [reply _projects/2019/juliaastro/astroimages.md]: One doesn't ensure the other (I suppose there are more astronomy students unfamiliar with FITS than the ones familiar enough for the project).
 But it's @dpshelio's call, I'm happy to have this merged as is or with removing the astronomy here.

Comment (giordano) [reply _projects/2019/juliaastro/astroimages.md]: Changed to "familiarity with FITS file", which is the real requirement

**Comment by bsipocz** [comment]
Thanks @giordano!

**Pull request merged**